recipes/pack.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
```

Comment:
What is this for?

Reply:
It's for packing the recipe, so one can simply deploy it on a host and enable the installation.